lib/WebService/Slack/WebApi/Dnd.pm

```diff
@@ -0,0 +1,28 @@
+package WebService::Slack::WebApi::Dnd;
```

Comment:
Please add two tests just like the others.

- https://github.com/mihyaeru21/p5-WebService-Slack-WebApi/blob/master/t/01_web_api.t#L26
- https://github.com/mihyaeru21/p5-WebService-Slack-WebApi/blob/master/t/03_call_methods.t#L10